### PR TITLE
generated models

### PR DESCRIPTION
**Pull request by alexberthon**
Generated models: User (with Devise), Pigeon, Rental, Review, PayloadType

### CODE REVIEW

File: app/models/pigeon.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Pigeon < ApplicationRecord
+  belongs_to :user
```
Comment (Vnt0n) [inline app/models/pigeon.rb:2]: has_many rentals

File: app/models/rental.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class Rental < ApplicationRecord
+  belongs_to :user
+  belongs_to :pigeon
+  belongs_to :payload_type
```
Comment (Vnt0n) [inline app/models/rental.rb:4]: has_many reviews

File: app/models/user.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (Vnt0n) [inline app/models/user.rb:5]: has_many pigeons / rentals / reviews

File: app/models/payload_type.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+class PayloadType < ApplicationRecord
+end
```
Comment (Vnt0n) [inline app/models/payload_type.rb:2]: has_many rentals

Comment (alexberthon) [reply app/models/payload_type.rb:3]: done

File: app/models/user.rb (hunk 3 above)
Comment (alexberthon) [reply app/models/user.rb:5]: done

File: app/models/rental.rb (hunk 2 above)
Comment (alexberthon) [reply app/models/rental.rb:4]: done

File: app/models/pigeon.rb (hunk 1 above)
Comment (alexberthon) [reply app/models/pigeon.rb:2]: done

**Pull request merged**